data/hai/unfettered 2 war preparations.txt

```diff
@@ -1056,9 +1057,12 @@ mission "<uhai> 15: Modified Beetle"
 					"outfit (all): Ionic Turret Prototype" > "turrets owned"
 			branch normal
 				not "unfettered rep: Mera"
-			`	"Since this was partly a test-run, what suggestions would you make to improve the design, before I finally give this ship to you?"`
+			`	"Since this was partly a test-run, what suggestions would you make to improve the ship, so I can make changes to its outfits, before I finally give this ship to you?"`
+			label suggestions
 			choice
 				`	"The ship is fine, but I could use an extra outfit to modify the loadout myself."`
```

Comment:
>"The ship is fine, but I could use an extra outfit to modify the loadout myself."

What does this mean? Do you want extra outfit space? One more of a specific outfit? One random outfit lying around?